src/Grpc/test/InteropTests/InteropTestsFixture.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace InteropTests
+{
+    public class InteropTestsFixture : IDisposable
```

Comment:
Why isn't the fixture in the `InteropTests.Infrastructure` namespace?

Reply:
Moved